conf/config.yaml.sample

```diff
@@ -69,7 +69,7 @@ cortx:
       - tcp://hax.seagate.com:28000
   csm:
     auth_admin: admin
-    csmauth_secret: auth-secret
+    auth_secret: csmauth-secret
```

Comment:
Please change this:
`auth_secret: csm-auth-secret`

Reply:
Done.